### DISCUSSION TITLE
Will this support framework 4.7 library

### DISCUSSION THREAD

**Discussion by amitkpal83** [nx-dotnet/nx-dotnet#573] [General]
Hi,

Will this support framework 4.7 version library. We are having project where lot of libraries are present in this version. Also UI in webforms and react . So is it supported and how much time it will take import all these?

Regards,
Amit

**Comment by AgentEnder** [comment]
This library only currently supports .NET Core, which is now just .NET. This includes versions .NET Core 3.1 -> .NET 7.

.NET framework is no longer being updated or maintained by Microsoft. The overall recommendation would be to migrate the code base to the latest versions of .NET, and then start using the plugin.

In the interim, I believe @bcallaghan-et may have some experience getting things to work with .NET Framework projects. See: https://github.com/nx-dotnet/nx-dotnet/issues/570#issuecomment-1329456791

**Comment by bcallaghan-et** [comment]
This plugin uses the dotnet CLI under the hood. While we have no plans to support legacy .NET Framework projects, the dotnet CLI does have some support for them. Many of the basic executors do work with .NET Framework. Your mileage may vary. You'll have better luck if your Framework projects use the new SDK-style of project file, rather than the older MSBuild style.
